security/bouncycastle-fips/bcfips/src/test/java/io/quarkus/ts/security/bouncycastle/fips/DisabledOnFipsAndNative.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.ts.security.bouncycastle.fips;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@Inherited
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@ExtendWith(DisabledOnFipsAndNativeCondition.class)
+public @interface DisabledOnFipsAndNative {
```

Comment:
I suggest to create a task to move this and `condition` into the framework

Reply:
https://github.com/quarkus-qe/quarkus-test-framework/issues/1372